cfn-resources/private-endpoint-service/test/inputs_1_create.template.json

```diff
@@ -0,0 +1,5 @@
+{
+    "ProjectId" : "",
+    "Region" : "",
+    "Profile": "default"
```

Comment:
Is there a reason contract test has very few properties? We should add all possible fields to avoid regressions in the future.